auth/src/android/credential_android.cc

```diff
@@ -441,6 +449,50 @@ Credential EmailAuthProvider::GetCredential(const char* email,
   return cred;
 }
 
+//
+// PhoneAuthCredential methods
+//
+PhoneAuthCredential::PhoneAuthCredential() {}
```

Comment:
Set `impl_` to `nullptr` and `error_code_` to `kAuthErrorNone;`.

The initial value for a pointer is not necessary to always be `nullptr` :(

Similarly in `credential_ios.mm`

Reply:
Ah, this has been done in `Credential` already. nvm

Reply:
I think you need to call parent constructor like 
```
PhoneAuthCredential::PhoneAuthCredential() : Credential() {}
```

Reply:
The compiler call's the default constructor of a parent class automatically.

Reply:
Ah, you are right. 
Seems like `Parent()` will always be called by default from ANY variation of child constructor. That is, `Child(const Child&)` would call `Parent()` by default if not specified.